src/pdm_packer/env.py

```diff
@@ -14,7 +15,14 @@
 except ImportError:
     from pdm.models.environment import Environment as BaseEnvironment
 
-IN_PROCESS_SCRIPT = Path(__file__).with_name("_compile_source.py")
+
+def get_IN_PROCESS_SCRIPT():
+    try:
+        return (
+            importlib.resources.files("pdm_packer") / "_compile_source.py"
+        ).as_file()
+    except:
```

Comment:
Better to use a specific exception, or `sys.version_info >= (3, 9)`

Reply:
donne... did catch a mistake which made the code use the depricated path without me noticing...